examples/lock-app/nrfconnect/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
-cmake_minimum_required(VERSION 3.13.1)
+cmake_minimum_required(VERSION 3.23)
```

Comment:
Why do we need 3.23 version? That can be a problem, as recently we agreed on recommending 3.21.2. As Zephyr doesn't work with 3.22.